modules/test/playwright/tests/commerce/commerce-product-content-web/productCard.spec.ts

```diff
@@ -256,3 +256,102 @@ test('COMMERCE-6193. As a buyer, I want the first selectable quantity of a produ
 		);
 	}
 });
+
+test('LPD-25497 Users should not be able to instantly add to cart for product card if options exist', async ({
+	apiHelpers,
```

Comment:
We can keep this test, but wouldn't a headless API test be more useful for the purpose of this?

We need to test what happens when you don't provide required options in headless etc.

Reply:
FYI, this test will need to be fixed depending on your answer on https://github.com/brianikim/liferay-portal/pull/200/commits/7c5e7b69144269db9c4ce4f1f8acda544f3581c5#r1980672376